packages/wrangler/src/secret/index.ts

```diff
@@ -323,7 +339,7 @@ export const secret = (secretYargs: CommonYargsArgv) => {
  * Remove trailing white space from inputs.
  * Matching Wrangler legacy behavior with handling inputs
  */
-function trimTrailingWhitespace(str: string) {
+export function trimTrailingWhitespace(str: string) {
```

Comment:
a duplicate of this fn is declared [here](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/pages/secret/index.ts#L30-L32) as well. Should we consider moving the fn to  a utilities file, and import it in both wrangler proper and pages?

Reply:
We should, moved!